### PR TITLE
Fix protobuf run dependency on old libignition-msgs5 Windows packages (yaml version)

### PR DESCRIPTION
**Pull request by traversaro**
Fix https://github.com/conda-forge/libignition-msgs1-feedstock/issues/30
Fix https://github.com/gazebosim/gazebo-classic/issues/3339 

Checklist

* [x] Used a static YAML file for the patch if possible ([instructions](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/blob/main/recipe/patch_yaml/README.md)).
* [x] Only wrote code directly into `generate_patch_json.py` if absolutely necessary.
* [x] Ran `python show_diff.py` and posted the output as part of the PR.
* [x] Modifications won't affect packages built in the future. 

<!-- Put any other comments or information here --!>

### CODE REVIEW

**Comment by beckermr** [comment]
Nice! FYI `artifact_in` can be a list of strings so you can combine some of the patches if you'd like!

**Comment by traversaro** [comment]
> Nice! FYI `artifact_in` can be a list of strings so you can combine some of the patches if you'd like!

Cool! I just did it.

File: recipe/patch_yaml/libignition-msgs1.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+if:
+  subdir_in: win-64
+  artifact_in: libignition-msgs5-5.1.0-h21ff451_1.tar.bz2
+  timestamp_lt: 1611218248868
+then:
+  - add_depends: "libprotobuf >=3.11.4,<3.12.0a0"
+---
+if:
+  subdir_in: win-64
+  artifact_in: libignition-msgs5-5.1.0-h21ff451_0.tar.bz2
```
Comment (beckermr) [inline recipe/patch_yaml/libignition-msgs1.yaml:44]: Did you want a timestamp here?

Comment (traversaro) [reply recipe/patch_yaml/libignition-msgs1.yaml:44]: Yes, thanks for noticing!

**Comment by beckermr** [comment]
Comments on the api or user experience with the yaml here are appreciated!

**Comment by traversaro** [comment]
> Comments on the api or user experience with the yaml here are appreciated!

It is definitely a big improvements over editing the big python script. I remember that the first time I had to modify the big python script it took me some time to understand were I was supposed to change, while with yaml and one file for feedstock everything is much more clear.

**Comment by beckermr** [comment]
Great!

Paste in the diff when you get a chance and then we can merge if it looks OK.

**Pull request merged**